addons/regency_shopsite/controllers/overlay_template_page.py

```
@@ -347,3 +349,22 @@ def overlay_template_price_list(self, overlay_template_id, **kwargs):
         return {
             'priceList': self._get_overlay_template_price_list(overlay_template_id),
         }
+
+    @http.route(['/shop/area_image/<model("overlay.product.area.image"):overlay_product_area_image_id>'],
+                type='http', auth="user", website=True)
+    def overlay_product_area_image(self, overlay_product_area_image_id):
+        if not overlay_product_area_image_id:
+            return NotFound()
```

Raise instead of return.